frontend/src/views/login-form-view.js

```diff
@@ -0,0 +1,68 @@
+import {html, PolymerElement} from '@polymer/polymer/polymer-element.js';
+import '@vaadin/vaadin-text-field/vaadin-text-field.js';
+import '@vaadin/vaadin-text-field/vaadin-password-field.js';
+import '@vaadin/vaadin-button';
+
+/**
+ * `login-form-view`
+ *
+ * LoginFormView element.
+ *
+ * @customElement
+ * @polymer
+ */
+class LoginFormView extends PolymerElement {
+
+    static get template() {
+        return html`
+            <style include="common-styles">
+                :host {
+                    display: block;
+                }
+                
+                .wrapper {
+                    margin: 0 auto;
+                    max-width: 400px;
+                    padding: var(--lumo-space-xl) var(--lumo-space-l) 0;
+                }
+                
+                .card {
+                    box-shadow: 0 0 0 1px var(--lumo-contrast-5pct), var(--lumo-box-shadow-s);
+                    border-radius: var(--lumo-border-radius-m);
+                    background: var(--lumo-base-color) linear-gradient(var(--lumo-tint-5pct), var(--lumo-tint-5pct));
+                    padding:  var(--lumo-space-m);
+                    
+                    display: flex;
+                    flex-direction: column;
+                }
+                
+                h3 {
+                    margin-top: var(--lumo-space-s);
+                }
+                
+                .card vaadin-button {
+                    margin: var(--lumo-space-m) 0;
+                }
+                
+                #forgotPasswordButton {
+                    margin: 0;
+                }
+            </style>
+            <div class="wrapper">
+                <div class="card">
+                    <h3>Log in</h3>
+                    <vaadin-text-field id="usernameField" label="Username" autocomplete="username" required></vaadin-text-field>
```

Comment:
At least 1Password works to fill these fields as it is committed. Tested this in the deployed version at https://labs.vaadin.com/layout-examples/login-form just now.

We could also link to `vaadin-login` repo so the user knows about it (e.g. from the front page or from this example), but the point of this example is to show how to create a similar looking layout (login is just an example here, it could be any other form too). Also `vaadin-login-form` is not really customizable atm. so it wouldn't really serve the purpose of this project to use it here.

Reply:
OK to keep the view as it is, as I mentioned.

We should have a list of "known issues" with the real password managers that caused us to end up with the current structure of `vaadin-login`(apart from the [requirement by Joonas](https://github.com/vaadin/vaadin-login/issues/8#issuecomment-451605239))

Reply:
Ok. I will keep it as is for now.